cpu/nrf5x_common/radio/nrfmin/nrfmin_gnrc.c

```diff
@@ -163,8 +163,8 @@ static gnrc_pktsnip_t *gnrc_nrfmin_recv(gnrc_netif_t *dev)
     if (nrfmin->dst_addr == NRFMIN_ADDR_BCAST) {
         netif->flags |= GNRC_NETIF_HDR_FLAGS_BROADCAST;
     }
-    netif->lqi = 0;
-    netif->rssi = 0;
+    netif->lqi = GNRC_NETIF_HDR_NO_LQI;
+    netif->rssi = GNRC_NETIF_HDR_NO_RSSI;
```

Comment:
Now I wonder: doesn't `gnrc_netif_hdr_init()` already take care of that?

Reply:
Yes, it does. I thought it wasn't calling it, as it previously also initialized those values by hand. I will drop them, but maybe as a separate commit before this one. So if this needs reverting, at least the cleanup can remain.

Reply:
Done